packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -1465,6 +1467,35 @@ void main() {
       const _MediaQueryAspectCase(MediaQuery.maybeDisplayFeaturesOf, MediaQueryData(displayFeatures: <DisplayFeature>[DisplayFeature(bounds: Rect.zero, type: DisplayFeatureType.unknown, state: DisplayFeatureState.unknown)])),
     ]
   ));
+
+  group('Linear TextScaler', () {
```

Comment:
Nit: If you do end up creating a separate text_scaler.dart file,  you could put these tests in a new text_scaler_test.dart file.  But it's fine either way.

Reply:
Looks like the TextScaler was moved to its own file. Let's move these tests to their own file as well since they don't relate much to media_quqery_test.dart.